SStorageStuff/Config/recipes.xml

```diff
@@ -9,6 +9,11 @@ recipes.xml
 
 <configs>
     <append xpath="/recipes">
+        <recipe name="cntCabinetBottomPlayer" count="1">
+            <ingredient name="resourceNail" count="15"/>
+            <ingredient name="resourceWood" count="24"/>
+        </recipe>
+
```

Comment:
Not necessary at this point, but once there are several cupboard shapes I think it would be better to make a variant helper with a single recipe; you may already have that planned

Reply:
Agreed.